bld/namelist_files/use_cases/hist_geoschem_nudged.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0"?>
+
+<namelist_defaults>
+
+<!-- Settings in this file were adapted from hist_trop_strat_vbs_cam6.xml -->
```

Comment:
Should this be hist_trop_strat_nudged_cam6.xml?

Reply:
I kept this comment from `hist_geoschem.xml` but yes, this is the same in both `hist_trop_strat_nudged_cam6.xml` and `hist_trop_strat_vbs_cam6.xml`. I can update the comment if needed?

Reply:
I prefer referencing the nudged file so it's clear what the analogous file is for all settings, including the nudged section.

Reply:
Thanks, references fixed. I noticed I actually added the referencing to `hist_trop_strat_nudged_cam6.xml` below where I added the nudged settings, but I updated the file header as well for clarity.

Reply:
Thanks!